Sources/Playbook/Components/Selectable Card/PBSelectableCard.swift

```diff
@@ -174,28 +182,46 @@ extension PBSelectableCard {
       separatorView
       Text(text)
         .padding(.horizontal, isSelected ? cardPadding : cardPadding + 1)
-        
     }
     .padding(.vertical, -4)
   }
   func radioInputView(_ text: String) -> some View {
     HStack(spacing: Spacing.none) {
-      PBRadio(
-        items: [
-          PBRadioItem(radioItem),
-        ],
-        orientation: .vertical,
-        selected: $isRadioSelected
+      Button("") {
+        radioId = id
+      }
+      .buttonStyle(
+        radioButtonStyle
       )
-      .padding(cardPadding)
+      .onChange(of: radioId, perform: { _ in
+        if String(describing: radioId) == cardText {
+          isSelected = true
+        } else {
+          isSelected = false
+        }
+      })
+      .padding(.horizontal, 10)
       separatorView
       Text(text)
+        .padding(cardPadding)
+        .padding(.horizontal, isSelected ? padding - 1 : 0)
+      
     }
+    .padding(.vertical, -4)
+  }
+  var radioButtonStyle: PBRadioButtonStyle {
+    PBRadioButtonStyle(
+      subtitle: nil,
+      isSelected: isSelected,
+      textAlignment: .horizontal,
+      padding: padding,
+      errorState: false
+    )
   }
   var separatorView: some View {
     PBSectionSeparator(orientation: .vertical, margin: 0) {}
```

Comment:
What about replace the section separator to a Divider()? The SectionSeparator don't allow much customizations, that's why it keeps a gray line when the card is selected.